src/insns/csrr_32bit.adoc

```diff
@@ -30,31 +30,52 @@ endif::[]
 Synopsis::
 CSR access (CSRRWI, CSRRS, CSRRSI, CSRRC, CSRRCI) 32-bit encodings
 
-Register Source Mnemonics::
-`csrr[s|c] rd, rs1, csr`
-
-Immediate Source Mnemonics::
-`csrr[w|s|c]i rd, imm, csr`
+Mnemonics for accessing capability CSRs at CLEN-wide aliases::
```

Comment:
Presumably this actually works for all CSR numbers, you just get null cap metadata if you use a non CLEN-wide CSR? Or would it give an assembly-time error?

Reply:
The instruction opcode is the same in _all_ cases. The idea is instead that the machine infers what's the size of the access based on the CSR number that you are accessing. For example, if you access mtvec, an XLEN value, then your assembly would print *x* registers, but if you access mtvec*c*, a CLEN value at a different CSR number, then your assembly would print *c* registers.

Reply:
Yeah that's what I mean, it seems a little weird to say "this one is for CLEN-wide aliases" when it would actually work for all CSRs.

Reply:
This is just about the assembly instead the actual encoding

Reply:
Ok. I can't resolve this conversation because Github's "Resolve conversation" button only seems to appear during a solar eclipse, but consider it resolved!

Reply:
Ah only the person that opened the PR or people with Write access to the repo can resolve conversations.